src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java

```diff
@@ -26,12 +27,27 @@
 import stepsdesign.BeforeActions;
 import java.io.IOException;
 import java.nio.file.Paths;
+import java.util.Base64;
 
 /**
  * Represents Test Setup and Clean up hooks.
  */
 public class TestSetupHooks {
-  private static boolean firstFileSinkTestFlag = true;
+  public static String testdata_Group = "";
```

Comment:
use camel case in the naming instead of underscore. (testdata_Group -> groupResponse)